src/backend/src/utils/design-review.utils.ts

```diff
@@ -0,0 +1,12 @@
+import { sendMessage } from '../integrations/slack';
+
+export const sendSlackCRReviewedNotification = async (slackId: string, designReviewId: string) => {
+  if (process.env.NODE_ENV !== 'production') return; // don't send msgs unless in prod
+  const msgs = [];
```

Comment:
We usually don't return the messages from these functions, but if you would like to for error handling or something, you should be able to just return the one instead of having it in an array